src/main/java/services/impl/FormRecordProcessorImpl.java

```diff
@@ -112,6 +139,13 @@ public static void purgeCases(UserSqlSandbox sandbox) {
         log.info(String.format(
                 "Purged [%d Case, %d Ledger] records in %dms",
                 removedCaseCount, removedLedgers, taken));
-
+        datadogStatsDClient.recordExecutionTime(
+                Constants.DATADOG_TIMINGS,
+                taken,
+                "domain:" + restoreFactory.getDomain(),
+                "username" + restoreFactory.getWrappedUsername(),
+                "request:" + "case-purge",
```

Comment:
Should we change this to something like "action" instead of "request"? Wherever we use "request" in formplayer datadog metrics it usually means the url that was hit; I think using that word now to mean essentially which function call we're timing is confusing. (The same comment would apply to other uses of "request" below as well.)